private/model/api/example.go

```diff
@@ -251,14 +251,7 @@ import (
 var _ time.Duration
 var _ strings.Reader
 var _ aws.Config
-
-func parseTime(layout, value string) *time.Time {
-	t, err := time.Parse(layout, value)
-	if err != nil {
-		panic(err)
-	}
-	return &t
-}
+var _, _ = protocol.ParseTime("unixTimestamp", "2016-09-27T15:50Z")
```

Comment:
Yes because, some services don't have the timestamp field in them and the "protocol" package would be imported and not used

Reply:
Ah i see, in that you the template doesn't need to actually call the function, all it needs is a reference to something within the package.  In this case a function pointer is good enough. Similar to how the other types are done above this.

```go
var _ protocol.ParseTime
```

Since functions can be used as values, the template only needs to have an anonymous variable referring to the function as a value.

Reply:
Oh ok, that makes sense!